test/include/dlaf_test/matrix/util_generic_blas.h

```diff
@@ -105,7 +105,7 @@ auto getLeftTriangularSystem(blas::Uplo uplo, blas::Op op, blas::Diag diag, T al
 ///      = (X_ij * op(A)_jj + (kk-1) * gamma) / alpha,
 /// where gamma = (j+1) / (i+2) * exp(I*(i+2*j)),
 ///       kk = j+1 if op(a) is an upper triangular matrix, or
-///       kk = m-j if op(a) is an upper triangular matrix.
+///       kk = m-j if op(a) is an lower triangular matrix.
```

Comment:
```suggestion
///       kk = n-j if op(a) is an lower triangular matrix.
```
And in line 90 it should be: `A(n x n)`